# DP: updates from the 10 branch upto 20200507 (6032350c350).

LANG=C git diff --no-renames --src-prefix=a/src/ --dst-prefix=b/src/ \
	6e6e3f144a33ae504149dc992453b4f6dea12fdb 6032350c350b72859f1280e0eb13c8af4c522422 \
	| awk '/^diff .*\.texi/ {skip=1; next} /^diff / { skip=0 } skip==0' \
	| grep -v -E '^(diff|index)'

--- a/src/gcc/BASE-VER
+++ b/src/gcc/BASE-VER
@@ -1 +1 @@
-10.1.0
+10.1.1
--- a/src/gcc/ChangeLog
+++ b/src/gcc/ChangeLog
@@ -1,3 +1,7 @@
+2020-05-07  Jakub Jelinek  <jakub@redhat.com>
+
+	* BASE-VER: Set to 10.1.1.
+
 2020-05-07  Release Manager
 
 	* GCC 10.1.0 released.
